src/chunk_constraint.c

```diff
@@ -80,31 +80,29 @@ chunk_constraints_expand(ChunkConstraints *ccs, int16 new_capacity)
 }
 
 static void
-chunk_constraint_choose_name(Name dst, bool is_dimension, int32 dimension_slice_id,
-							 const char *hypertable_constraint_name, int32 chunk_id)
+chunk_constraint_dimension_choose_name(Name dst, int32 dimension_slice_id)
 {
-	if (is_dimension)
-	{
-		snprintf(NameStr(*dst), NAMEDATALEN, "constraint_%d", dimension_slice_id);
-	}
-	else
-	{
-		char constrname[100];
-		CatalogSecurityContext sec_ctx;
+	snprintf(NameStr(*dst), NAMEDATALEN, "constraint_%d", dimension_slice_id);
+}
+
+static void
+chunk_constraint_choose_name(Name dst, const char *hypertable_constraint_name, int32 chunk_id)
+{
+	char constrname[100];
```

Comment:
why is this hardcoded 100, shouldnt this be NAMEDATALEN anything past that would get truncated anyway and might even create unrecoverable errors if we start creating names that get truncated because identifier names are no longer unique and chunk creation would fail as a result

Reply:
It is a move of the original code, which contained this magic number. I can look into fixing it, but it is not something I introduced.

Reply:
`NAMEDATALEN` should be correct here. So changed to it.